package.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "chat",
-    "displayName": "Live Share Chat",
-    "description": "Chat with your team while you collaborate over code using VS Live Share",
-    "version": "0.31.0",
+    "displayName": "Chat",
```

Comment:
@lostintangent, @fubaduba: Also the displayName, description and version may need to be adjusted